### PR TITLE
Add lock and unlock methods and virtuals

### PR DESCRIPTION
**Pull request by chenrozenes**

### CODE REVIEW

File: src/statics.js [hunk 1]
```diff
@@ -29,7 +30,13 @@ export default function (config) {
                   if (!!throwError) throw Errors.LOGIN_FAILED;
                   return false;
                 }
-                
+
+                // Check if locked
+                if (user.isLocked) {
```
Comment (rom10) [inline src/statics.js]: I prefer to add !! before
`if (!!user.isLocked) {`

Comment (rom10) [inline src/statics.js]: I prefer to add !! before
`if (!!user.isLocked) {`

File: src/user.js [hunk 2]
```diff
@@ -85,6 +85,14 @@ export default function UserPlugin(schema, options) {
     return _.get(this.emails, '[0].verified');
   });
 
+  schema.virtual('isLocked').set(function(isLocked) {
```
Comment (rom10) [inline src/user.js]: change to locked

File: src/statics.js (hunk 1 above)
Comment (chenrozenes) [inline src/statics.js]: okkkk

**Pull request merged**